drivers/can/can_mcan.c

```diff
@@ -76,6 +48,27 @@ int can_mcan_write_reg(const struct device *dev, uint16_t reg, uint32_t val)
 	return err;
 }
 
+int can_mcan_read_mram(const struct device *dev, uint16_t offset, void *dst, size_t len)
```

Comment:
Woulit be an option to put them in the header and mark them inline?

Reply:
Good idea. I have moved the implementations to the header file and inlined them.